lib/TeamcityReporter.js

```diff
@@ -33,10 +33,19 @@ class TeamcityReporter {
     item(err, args) {
         if (!this.currItem.passed) {
             const msg = this.escape(this.currItem.failedAssertions.join(", "));
-            const responseCode = (this.currItem.response && this.currItem.response.responseTime) || "-";
+            const responseCode = this.currItem.response.code;
             const reason = (this.currItem.response && this.currItem.response.reason()) || "-";
             const details = this.escape(`Response code: ${responseCode}, reason: ${reason}`);
-            console.log(`##teamcity[testFailed name='${this.currItem.name}' message='${msg}' details='${msg} - ${details}']`);
+            console.log(`##teamcity[testFailed name='${this.currItem.name}' details='${msg} - ${details}']`);
```

Comment:
`message` is intended as a succinct description of the failure and `details` is a more verbose explanation. Neither is redundant.

Reply:
I disagree on this.  Below is the teamcity log example if keep it as it was,   
![image](https://user-images.githubusercontent.com/10079887/34801344-51f71744-f6bc-11e7-887a-6c457ee98d53.png)
If take the fix solution, the team city log will remove the log info in the first line, the following line (verbose one) is including the same duplicated log as the above line.
![image](https://user-images.githubusercontent.com/10079887/34801601-ab648d9c-f6bd-11e7-9ea0-6c89bb977837.png)